canvas_oauth/urls.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import url
-from oauth import oauth_callback
+from django.urls import re_path
+from .oauth import oauth_callback
 
 urlpatterns = [
-    url(r'^oauth-callback$', oauth_callback, name='canvas-oauth-callback'),
+    re_path(r'^oauth-callback$', oauth_callback, name='canvas-oauth-callback'),
```

Comment:
It is possible to just use `path` rather than `re_path` since there is no regular expression being evaluated? something like: `path('oauth-callback/', oauth_callback, name='canvas-oauth-callback')`

Reply:
Yes, definitely. The regex seems unnecessary here, so no reason not to use `path()`. I'll go ahead and update that.